### PR TITLE
Add return instruction to nucleotide-count

### PR DESCRIPTION
**Pull request by DavidMatalik**
This PR is related to issue #1474

### CODE REVIEW

**Review by junedev — CHANGES_REQUESTED** [review]
Please have a closer look at the issue description. The output in the JavaScript version of the exercise is a string that contains 4 numbers separated by spaces, not an object.

**Comment by DavidMatalik** [comment]
> Please have a closer look at the issue description. The output in the JavaScript version of the exercise is a string that contains 4 numbers separated by spaces, not an object.

Thanks for pointing that out to me. I 've read something about squashing commits. What is the best way to proceed in this situation here? Amending the commit locally and force pushing it?

**Comment by SleeplessByte** [comment]
Hi @DavidMatalik . You don't need to squash or rewrite your commits. Continue as usual. What we'll do when we merge your PR is "squash and merge", which will rewrite all your commits (including the rejected ones) into a single commit, removing the history.

File: exercises/practice/nucleotide-count/.docs/instructions.append.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Instructions append
+
+The output should be a String having 4 numbers separated by spaces. Each number represents a count for A, C, G and T in this order.
+
+For example:
+```javascript
+NucleotideCounts.parse('GATTACA') -> '3 1 1 2'
```
Comment (junedev) [inline exercises/practice/nucleotide-count/.docs/instructions.append.md]: There was another PR that changed this signature. I updated this for you and also tweaked the sentence above a little.

**Review by junedev — APPROVED** [review]

**Pull request merged**